File: app/src/main/java/com/example/moetaz/movieapp/Activity/DetailActivity.java
package com.example.moetaz.movieapp.activity;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.example.moetaz.movieapp.fragment.DetailFragment;
import com.example.moetaz.movieapp.R;

/**
 * Created by Moetaz on 11/4/2016.
 */

public class DetailActivity extends AppCompatActivity {
      @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .add(R.id.fdetail, new DetailFragment())
                    .commit();
        }

    }


}

File: app/src/main/java/com/moetaz/popularmoviesapp/ui/activities/DetailActivity.kt
package com.moetaz.popularmoviesapp.ui.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.moetaz.popularmoviesapp.R
import com.moetaz.popularmoviesapp.ui.fragments.DetailFragment

class DetailActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                .add(R.id.fdetail , DetailFragment()).commit()
        }
    }
}
